static/app/types/index.tsx

```diff
@@ -103,6 +103,9 @@ declare global {
      * See sentry/js/ads.js for how this global is disabled.
      */
     adblockSuspected?: boolean;
+
+    //TODO: improve typing
+    SentryApp: Record<string, any>;
```

Comment:
surprised we have no typings on this but I need a type in order to add `modalFocusTrap` when `window.SentryApp` already exists

Reply:
So as part of frontend deploys, we will need to make a breaking change where `window.SentryApp` will no longer be guaranteed to be available on document ready. Can we:

1) Explicitly type what's required in sandbox
2) Add some comments why those properties are required
3) In demo mode, if you need access to `SentryApp`, you will want to use `window.__onSentryInit`, e.g.

```
window.__onSentryInit.push({
  name: 'onReady',
onReady: (globals) => {
  // do things here
}
})
```

Reply:
@billyvg updated the PR with this